recipe/migrations/cuda110.yaml

```
@@ -0,0 +1,28 @@
+migrator_ts: 1601612527
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
```

```suggestion
    1
  override_cbc_key:
   - cuda_compiler_stub
```

This change will allow this migrator to use the new bot functionality to migrate against compiler updates.